app/views/nodes/notes/_empty.html.erb

```diff
@@ -0,0 +1,34 @@
+<h1>This node doesn't have any notes yet</h1>
+
+<p>Add one by clicking on the button below or use the <i class="fa fa-plus"></i> sign next to the <strong>Notes</strong> heading in the sidebar
+
+<div class="dropdown offset2">
+  <%= link_to 'javascript:void(0)', class: 'btn dropdown-toggle', data: { name: 'notes', toggle: 'dropdown'} do %>
+    <i class="fa fa-plus"></i> Add note
+  <% end %>
+
+  <ul class="dropdown-menu" role="menu">
+    <li>
+      <%= link_to 'Empty (no template)',
+        new_node_note_path(@node),
+        :tabindex => '-1',
+        :data => {
+          :category_id => Category.default.id,
+          :title       => "Add note",
+        } %>
+    </li>
+
+    <li class="divider"></li>
+
+    <% if NoteTemplate.all.any? %>
+      <% NoteTemplate.all.each do |template| %>
+        <li>
+          <%= link_to template.name, "#{new_node_note_path(@node)}?template=#{template.id}", tabindex: '-1' %>
```

Comment:
can you use the syntax: `new_node_note_path(@node, template: template.id)` here?